frontend/src/util/validations.js

```diff
@@ -0,0 +1,19 @@
+const _ = require("lodash");
+
+function languageValidator(language) {
+  return _.includes(["hebrew", "english"], language);
+}
+
+function paramsValidator(entitiyName) {
```

Comment:
I don't understand what's the purpose of this function. I think you should rename it to make it more clear. And maybe also extract the array you have here to a const with a meaningful name.

Reply:
Regarding the rename of the function, rename the function to the `isX` pattern, since it's boolean